src/BenchmarksClient/Workers/SignalRWorker.cs

```diff
@@ -144,6 +157,24 @@ public SignalRWorker(ClientJob job)
                             }
                         }
                         break;
+                    case "echoIdle":
+
+                        if(_sendDelayTimer == null)
+                        {
+                            _sendDelayTimer = new Timer(async (timer) =>
+                            {
+                                if (!cts.IsCancellationRequested)
+                                {
+                                    for (var id = 0; id < _connections.Count; id++)
+                                    {
+                                        var time = await _connections[id].InvokeAsync<DateTime>("Echo", DateTime.UtcNow, cts.Token);
```

Comment:
remove the `cts.Token` from here, these are long running and the `cts` is per job

Reply:
Add a try catch around this to prevent unobserved exceptions, and log I guess